docs/docs/get-started/why-rill.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Why Rill?
+sidebar_label: Why Rill?
+sidebar_position: 10
+---
+
+Data teams often struggle with a fragmented stack: data lakes are cheap but slow, while data warehouses are fast but expensive and rigid. Engineering teams spend too much time managing ETL pipelines and answering ad-hoc requests, while business users are stuck with slow, static dashboards that don't allow for true exploration.
```

Comment:
> data warehouses are fast but expensive

I'm not sure this is the best characterization. Traditionally, data warehouses are cost-effective for large data volumes, and their performance is historically relatively slower than the high-performance OLAP category (ClickHouse, Druid, Pinot).